index.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+var aws = require('aws-sdk-promise');
+var lodash = require('lodash');
+var debug = require('debug')('multi-region-promised-aws');
+var Promise = require('promise');
+
+// TODO: allow for each region to have its own defaults and settings
+function MultiAWS(nameOrConstructor, config, regions) {
```

Comment:
Please add a doc comment like:
```
 /**
 * This class represents... bla.. .bla...
 *
 * config:  // AWS config...
 * etc....
 */
```

Anything like that... look at other code we have base.Entity has a lot of the comments in a consistent format.
We don't have docs generator for them (sadly) maybe one day we'll find something that actually works...